sdk/keyvault/keyvault-certificates/src/certificatesModels.ts

```diff
@@ -89,6 +88,36 @@ export interface CertificateClientInterface {
  */
 export type CertificateContentType = "application/pem" | "application/x-pkcs12" | undefined;
 
+/**
+ * The action that will be executed.
+ */
+export interface CertificatePolicyAction {
+  /**
+   * The type of the action. Possible values include: 'EmailContacts', 'AutoRenew'
+   */
+  actionType?: ActionType;
+}
+
+/**
+ * Action and its trigger that will be performed by Key Vault over the lifetime of a certificate.
+ */
+export interface LifetimeAction {
+  /**
+   * Percentage of lifetime at which to trigger. Value should be between 1 and 99.
+   */
+  lifetimePercentage?: number;
+  /**
+   * Days before expiry to attempt renewal. Value should be between 1 and validity_in_months
+   * multiplied by 27. If validity_in_months is 36, then value should be between 1 and 972 (36 *
+   * 27).
+   */
+  daysBeforeExpiry?: number;
+  /**
+   * The action that will be executed.
+   */
+  action?: CertificatePolicyAction;
```

Comment:
This should actually either be "EmailContacts" or "AutoRenew", not have a property that does that.